coderedcms/models/page_models.py

```diff
@@ -1502,7 +1555,7 @@ def create_submission_deleted_revision(sender, **kwargs):
         return
     submission = kwargs['instance']
     CoderedSubmissionRevision.create_from_submission(submission,
-                                              SubmissionRevision.DELETED)
+                                                     SubmissionRevision.DELETED)
```

Comment:
I don't think this is correct indentation... should probably just be 4 spaces like normal.